app/main.py

```diff
@@ -2,0 +1,33 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
```

Comment:
Code Style: It's a good practice to use type annotations for all function parameters and return types. You have done it for the `__init__` method, but missed it for the `__repr__`, `die`, `hide` and `bite` methods.